config/production.json

```diff
@@ -173,12 +172,6 @@
 		"subscriber-csv-upload": true,
 		"subscriber-importer": true,
 		"subscription-gifting": true,
-		"subscription-management-add-sites-modal": true,
-		"subscription-management-comments-view": true,
-		"subscription-management-pending-view": true,
-		"subscription-management/comments-list-controls": true,
-		"subscription-management/migrate-subscribers": true,
```

Comment:
This was not enabled on wpcalypso but was enabled everywhere else. Weird, seems unintended.